### PR TITLE
Display issue with flow chart on user-guide/concepts/installing-with-conda page

### PR DESCRIPTION
**Pull request by travishathaway**
### Description

Fixes: #13373 

I also took a little time to update the introduction. I think this a good thing, but I am happy to hear feedback.

### Checklist - did you ...

- [ ] ~Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?~
     
- [ ] ~Add / update necessary tests?~
- [x] Add / update outdated documentation?

### CODE REVIEW

File: docs/source/user-guide/concepts/installing-with-conda.rst [hunk 1]
```diff
@@ -2,41 +2,43 @@
...
-To install conda packages, in the terminal or an Anaconda Prompt, run::
-
-  conda install [packagename]
+  conda install <package>
 
+When conda installs a package, it is automatically added to your active
+environment. These packages are collections of files and directories
+that make up everything you need to use that particular
+library or software. For Python packages, these are primarily Python
+files that can be imported in other Python applications, but for compiled
```
Comment (kathatherine) [inline docs/source/user-guide/concepts/installing-with-conda.rst:15]: Imported into?

File: docs/source/user-guide/concepts/installing-with-conda.rst [hunk 2]
```diff
@@ -2,41 +2,43 @@
...
+software packages such as ``ffmpeg`` these are typically binary executables
```
Comment (kathatherine) [inline docs/source/user-guide/concepts/installing-with-conda.rst:16]:
```suggestion
software packages, such as ``ffmpeg``, these are typically binary executables
```

File: docs/source/user-guide/concepts/installing-with-conda.rst [hunk 3]
```diff
@@ -2,41 +2,43 @@
...
-    :align: right
-
 .. _installing-with-conda:
 
+Conda packages can be installed by running the following command::
 
-To install conda packages, in the terminal or an Anaconda Prompt, run::
-
-  conda install [packagename]
+  conda install <package>
```
Comment (kathatherine) [inline docs/source/user-guide/concepts/installing-with-conda.rst:9]: We should attempt to have a standard for placeholders in commands going forward. I'm not married to any one thing, so `<all-lowercase-words>` works for me. I've also used all-caps, e.g. `<PACKAGE>`.

Any opinions?

File: docs/source/user-guide/concepts/installing-with-conda.rst [hunk 4]
```diff
@@ -2,41 +2,43 @@
...
-* When you ``conda install`` a package that exists in a channel and has no dependencies, conda:
+Below is more precise overview of everything that happens during the installation
+process for a single package:
 
-  * Looks at your configured channels (in priority).
+* Currently configured channels (e.g. ``defaults`` or ``conda-forge``) are read in order of priority
+* Repodata for these configured channels will be downloaded and read
+* The repodata will be searched for the package starting with the highest priority channel first
+* Once the package is found, conda makes a separate download request and then installs it
+* This process will then repeat for each of the packages dependencies if there are any
```
Comment (kathatherine) [inline docs/source/user-guide/concepts/installing-with-conda.rst:31]:
```suggestion
* Repodata for these configured channels is downloaded and read
* The repodata is searched for the package, starting with the highest priority channel first
* Once the package is found, conda makes a separate download request and then installs it
* This process then repeats for each of the package's dependencies, if there are any
```
Suggestions for keeping the verb tense the same throughout, plus some punctuation edits

**Review by kathatherine — CHANGES_REQUESTED** [review]
Couple things. Thanks for the rewrite!

File: docs/source/user-guide/concepts/installing-with-conda.rst [hunk 5]
```diff
@@ -2,41 +2,43 @@
...
-and its dependencies---all with the single
-``conda install [packagename]`` command.
+.. admonition:: Note
 
-Read more about :doc:`conda environments and directory structure <../concepts/environments>`.
+    If you would like to learn more about how environments are structured,
+    head over to :doc:`conda environments<../concepts/environments>`.
 
-* When you ``conda install`` a package that exists in a channel and has no dependencies, conda:
+Below is more precise overview of everything that happens during the installation
```
Comment (beeankha) [inline docs/source/user-guide/concepts/installing-with-conda.rst:24]:
```suggestion
Below is a more precise overview of everything that happens during the installation
```

Comment (kathatherine) [reply docs/source/user-guide/concepts/installing-with-conda.rst:24]: Good catch!

File: docs/source/user-guide/concepts/installing-with-conda.rst (hunk 3 above)
Comment (travishathaway) [reply docs/source/user-guide/concepts/installing-with-conda.rst:9]: I like `<all-lowercase>`. I always get a little anxious when I SEE ALL CAPS!!! 🙀.

File: docs/source/user-guide/concepts/installing-with-conda.rst (hunk 1 above)
Comment (travishathaway) [inline docs/source/user-guide/concepts/installing-with-conda.rst:15]:
```suggestion
files that can be imported into other Python applications, but for compiled
```

File: docs/source/user-guide/concepts/installing-with-conda.rst (hunk 3 above)
Comment (dholth) [reply docs/source/user-guide/concepts/installing-with-conda.rst:9]: 🧢🧢🧢

Comment (kathatherine) [reply docs/source/user-guide/concepts/installing-with-conda.rst:9]: Works for me!

**Review by kathatherine — APPROVED** [review]
LGTM!

**Review by beeankha — APPROVED** [review]

**Pull request merged**